### PR TITLE
make ci tests runs use the job m2 repo

### PR DESCRIPTION
**Pull request by lhein**

### CODE REVIEW

File: editor/plugins/org.fusesource.ide.launcher.ui/src/org/fusesource/ide/launcher/ui/launch/ExecutePomActionSupport.java [hunk 1]
```diff
@@ -338,6 +341,14 @@ private ILaunchConfiguration createLaunchConfiguration(IContainer basedir, Strin
 			ILaunchConfigurationWorkingCopy workingCopy = launchConfigurationType.newInstance(null, launchConfigName);
 			workingCopy.setAttribute(MavenLaunchConstants.ATTR_POM_DIR, basedir.getLocation().toOSString());
 			workingCopy.setAttribute(MavenLaunchConstants.ATTR_GOALS, goal);
+			
+			String customRepoPath = getCustomRepositoryPath();
+			if (!Strings.isBlank(customRepoPath)) {
+				List<String> props = workingCopy.getAttribute(MavenLaunchConstants.ATTR_PROPERTIES, new ArrayList<String>());
+				props.add("-Dmaven.repo.local" + customRepoPath);
```
Comment (lhein) [inline editor/plugins/org.fusesource.ide.launcher.ui/src/org/fusesource/ide/launcher/ui/launch/ExecutePomActionSupport.java]: good point. will add a check.

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
that one needs to be rebased on latest master

**Comment by apupier** [comment]
I used this PR for the rebase work: https://github.com/jbosstools/jbosstools-fuse/pull/992

**Pull request closed without merging**